docs/source/install.rst

```diff
@@ -135,20 +135,36 @@ continue by reading the :ref:`usage documentation`.
 Installing from a git clone
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you want to follow the latest development or contribute to LiberTEM, you
-should install LiberTEM from a git clone:
+If you want to follow the latest development, you should install LiberTEM from
+a git clone:
 
 .. code-block:: shell
 
     $ git clone https://github.com/LiberTEM/LiberTEM
 
+Or if you wish to contribute to LiberTEM, follow these steps instead :
+
+#. Log into your `GitHub <https://github.com/>`_ account.
+
+#. Go to the `LiberTEM GitHub <https://github.com/liberteM/LiberTEM/>`_ home page.
+
+#. Click on the *fork* button:
+
+    ..  figure:: ./images/forking_button.png
+
```

Comment:
Perhaps better to link to GitHub's documentation on the topic? https://help.github.com/en/github/getting-started-with-github/fork-a-repo